grub2/install

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+
+target=$(uname --hardware-platform)
+
+if [ -z "$target" ] ; then
+  echo "no target platform"
+  exit 1
+fi
+
+case "$target" in
+  i?86 | x86_64 ) target=i386-pc ;;
+  ppc | ppc64 ) target=powerpc-ieee1275 ;;
+  s390x ) target=s390x-emu ;;
+esac
+
+echo "target = $target"
+
+err=0
+
+if [ -x /usr/sbin/grub2-install ] ; then
+  if [ -f /etc/default/grub_installdevice ] ; then
+    while read foo ; do
+      # ignore everything that doesn't look like a path
+      [ "${foo::1}" != "/" ] && continue
+      if [ -b "$foo" -o -f "$foo" ] ; then
+        ( set -x ; /usr/sbin/grub2-install --target="$target" --force --skip-fs-probe "$foo" ) || err=1
+      else
+        echo "$foo: not a block device"
+        err=1
+      fi
+    done </etc/default/grub_installdevice
+  else
+    echo "/etc/default/grub_installdevice: no such file"
+    err=1
+  fi
+else
+  echo "grub2-install: command not found"
+  err=1
+fi
+
+exit $err
```

Comment:
yeah, it was old option for lilo, which need reinit everytime its config changed. I think now only possible user is grub2 itself, when it need update stage1 code.